core/src/mindustry/ui/dialogs/ModsDialog.java

```diff
@@ -133,11 +144,121 @@ void setup(){
                             }, t2 -> Core.app.post(() -> modError(t2)));
                         });
                     }).margin(12f);
-                });
 
+                    t.row();
+
+                    t.button("$mod.featured.title", Icon.star, bstyle, () -> {
+                        try {
+                            dialog.hide();
+                            BaseDialog dialog2 = new BaseDialog("$mod.featured.dialog.title");
+                            dialog2.cont.table(table -> {
+                                table.left();
+                                table.image(Icon.zoom);
+                                searchBar = table.field(searchtxt, res -> {
+                                    searchtxt = res;
+                                    rebuildBrowser[0].run();
+                                }).growX().get();
+                            }).fillX().padBottom(4);
+
+                            dialog2.cont.row();
+
+                            dialog2.cont.pane(tablebrow -> {
+                                tablebrow.margin(10f).top();
+                                rebuildBrowser[0] = () -> {
+                                    tablebrow.clear();
+                                    String searchString = searchtxt.toLowerCase();
+                                    Core.net.httpGet("https://raw.githubusercontent.com/Anuken/MindustryMods/master/mods.json", response -> {
+                                        if (response.getStatus() != HttpStatus.OK) {
```

Comment:
The formatting in this file is incorrect. You're not following the [style guidelines](https://github.com/Anuken/Mindustry/blob/master/CONTRIBUTING.md#style-guidelines).

Reply:
I dont really know which im violating. If you are referring to tabs instead of 4 spaces i can change it, but i think everything else is good

Reply:
Oh okay, you meant if(cond) {}